catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml

```diff
@@ -21,29 +21,25 @@ spec:
       description: The kubernetes secret to use to authenticate to Pyxis
     - name: certified
       type: string
-      description: If set to true, the image will be marked as certified in its Pyxis entry
+      description: If set to true, the images will be marked as certified in their Pyxis entries
       default: "false"
     - name: tag
       type: string
       description: The tag to use when pushing the container image metadata to Pyxis
     - name: isLatest
       type: string
-      description: If set to true, the image will have a latest tag added with its Pyxis entry
+      description: If set to true, the images will have a latest tag added with their Pyxis entries
       default: "false"
-    - name: skopeoInspectFile
+    - name: snapshot
       type: string
-      description: The filename of the saved skopeo inspect output stored on the input workspace
-      default: "skopeo-inspect.json"
+      description: The snapshot in JSON format
   results:
     - name: containerImageID
-      description: ID of the created entry in Pyxis
-  workspaces:
-    - name: input
-      description: The workspace where the skopeoInspectFile is located
+      description: ID(s) of the created entries in Pyxis
```

Comment:
Again, I would suggest to remove the `()` since we should always treat this as plural IDs even if there is just one.